packages/models/src/utils.ts

```diff
@@ -0,0 +1,11 @@
+export const bigIntToDateOrUndefined = (
+  input: bigint | undefined
+): Date | undefined => (input ? new Date(Number(input)) : undefined);
+
+export const bigIntToDate = (input: bigint): Date => new Date(Number(input));
```

Comment:
We could just create one function that handles both input of type `bingint` and input of type `bigint | undefiend` using function overloads! 

```suggestion
export function bigIntToDate(input: bigint): Date;
export function bigIntToDate(input: bigint | undefined): Date | undefined;
export function bigIntToDate(input: bigint | undefined): Date | undefined {
  return input ? new Date(Number(input)) : undefined;
}
```

Reply:
I thought about keeping them separated to get a better overview of which field is required/optional, for example:
``` 
createdAt: dateToBigInt(input.createdAt),
updatedAt: dateToBigIntOrUndefined(input.updatedAt)
```

Reply:
I agree with @Carminepo2, using the overload can be helpful.
> I thought about keeping them separated to get a better overview of which field is required/optional, for example:
> 
> ```
> createdAt: dateToBigInt(input.createdAt),
> updatedAt: dateToBigIntOrUndefined(input.updatedAt)
> ```
In these cases I don't think it would be a problem actually it will make it easier to write the converters since you don't have to worry about which function to use.

Reply:
Updated